backend/src/core/routes/campaign.routes.ts

```diff
@@ -63,6 +68,40 @@ const updateCampaignValidator = {
  *            type: integer
  *            minimum: 0
  *            default: 0
+ *        - in: query
+ *          name: type
+ *          description: mode of campaigns to filter for
+ *          required: false
+ *          schema:
+ *            type: string
+ *            enum: [EMAIL, SMS, TELEGRAM]
+ *        - in: query
+ *          name: status
+ *          description: status of campaigns to filter for
+ *          required: false
+ *          schema:
+ *            type: string
+ *            enum: [Draft, Sending, Sent]
+ *        - in: query
+ *          name: name
+ *          description: name of campaigns to filter for
+ *          required: false
+ *          schema:
+ *            type: string
+ *        - in: query
+ *          name: sort_by
+ *          description: field used to sort campaigns
+ *          required: false
+ *          schema:
+ *            type: string
+ *            enum: [name, created_at, sent_at]
```

Comment:
What if I want to sort by more than one field? Can we use a `+created_at` `-created_at` system instead of having 2 diff parameters `sort_by` and `order_by`? (i.e. combining the order information right inside the values for `sort_by`)

Reply:
to deal with in future PR?

Reply:
This is an irreversible decision, we can't go back and revise it later after people have started integration using this API interface. It's gotta be decided and resolved right in this PR

Reply:
ohh I see your point — I don't think Rae will be working on this further (she's flying off tmr), think we should resolve this between us.

Reply:
As per discussed, we probably don't ever need sort by multiple fields with campaigns so just go with this design